### PR TITLE
Adds three new Christmas songs

### PR DESCRIPTION
**Pull request by Cartlord**
# Document the changes in your pull request

I've added three songs by Phyrnna ([Sisters of Snow Dissent](https://www.youtube.com/watch?v=Ri-Pmh5X8-c), [Sisters of Snow Assent](https://www.youtube.com/watch?v=W2Dyg4YY2VU), and [A First Snow with Friends](https://www.youtube.com/watch?v=VICEsWdd4Kk)) to the Christmas music playlist, because I really like them and I think they fit the season to an extent.

# Changelog

:cl:   
soundadd: Added three songs by Phyrnna to the Christmas music playlist.  
/:cl:

### CODE REVIEW

**Comment by JamieD1** [comment]
put the fecking urls in the description

**Comment by JamieD1** [comment]
https://polsy.org.uk/stuff/ytrestrict.cgi?ytid=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3D2n0KCYBKLhk&auth=06d210790ab583cab38eabbdf16ac3e8

Other two are fine

**Comment by Cartlord** [comment]
> put the fecking urls in the description

All three song names are YouTube hyperlinks.

**Comment by Cartlord** [comment]
> https://polsy.org.uk/stuff/ytrestrict.cgi?ytid=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3D2n0KCYBKLhk&auth=06d210790ab583cab38eabbdf16ac3e8
> 
> Other two are fine

Alright, changed the U.R.L. for that one. I checked using that website, and it's unrestricted.

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -599,7 +599,11 @@ Since Ramadan is an entire month that lasts 29.5 days on average, the start and
 		"https://www.youtube.com/watch?v=1twga61Kd14",	// Julia Ecklar - #1 - Christmas Time
 		"https://www.youtube.com/watch?v=imjMjnczqkU",	// Pete Gold - Ive Been a Bad Boy
 		"https://www.youtube.com/watch?v=aAkMkVFwAoo",   //Mariah Carey, All I Want For Christmas Is You
-		"https://www.youtube.com/watch?v=1HGQlV2AWGs"	// Phineas and Ferb - We Wish You a Merry Christmas
+		"https://www.youtube.com/watch?v=Ri-Pmh5X8-c",   //Phyrnna - Sisters of Snow Dissent
+		"https://www.youtube.com/watch?v=W2Dyg4YY2VU",   //Phyrnna - Sisters of Snow Assent
+		"https://www.youtube.com/watch?v=VICEsWdd4Kk",   //Phyrnna - A First Snow with Friends
+    "https://www.youtube.com/watch?v=1HGQlV2AWGs"	// Phineas and Ferb - We Wish You a Merry Christmas
```
Comment (JamieD1) [inline code/modules/holiday/holidays.dm:605]:
```suggestion
"https://www.youtube.com/watch?v=1HGQlV2AWGs"	// Phineas and Ferb - We Wish You a Merry Christmas
```

Comment (JamieD1) [inline code/modules/holiday/holidays.dm]:
```suggestion
		"https://www.youtube.com/watch?v=VICEsWdd4Kk",   //Phyrnna - A First Snow with Friends
		"https://www.youtube.com/watch?v=1HGQlV2AWGs"	// Phineas and Ferb - We Wish You a Merry Christmas
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**